test/integration/rules/filename-is-valid-accessible-name/filename-is-valid-accessible-name.html

```diff
@@ -0,0 +1,29 @@
+<!-- Pass -->
```

Comment:
No test for this example from the dos:

```html
<a href="https://www.w3.org/WAI/demos/bad/img/w3c.png" download
	>Download <img src="w3c.png" alt="w3c.png"
/></a>
```

Reply:
Yeah, this is up for debate with in the ACT Rule community. I think we ought to only look for the element and not its context for computing accessible name. That brings in an entire new spectrum. Leave this with me to discuss with concerned peeps.

Reply:
@WilcoFiers will appreciate your input on this.

Reply:
Spoke to @WilcoFiers. The rule will not compute accessible name outside of the context of the given node/ thyself. So for this example, `incomplete` will be returned.